apps/web-giddh/src/app/shared/header/components/account-add-new-details/account-add-new-details.component.ts

```diff
@@ -1653,5 +1686,129 @@ export class AccountAddNewDetailsComponent implements OnInit, OnChanges, AfterVi
             });
         }
     }
+
+    /**
+     * Open Dialog for Bulk Add
+     *
+     * @memberof AccountAddNewDetailsComponent
+     */
+    public openBulkAddDialog(): void {
+        if (this.addAccountForm.get('currency')?.value !== this.companyCurrency) {
+            this.isForeignCurrecny = true;
```

Comment:
```suggestion
            this.isForeignCurrecny = this.addAccountForm.get('currency')?.value !== this.companyCurrency;
```

write this and remove if else